src/Database/Table/SqlBuilder.php

```diff
@@ -262,6 +262,7 @@ public function importConditions(self $builder): void
 	public function importGroupConditions(self $builder): bool
 	{
 		if ($builder->having) {
+            $this->select = $builder->select;
```

Comment:
@dg this is fix for mysql, which needs field to be in group by, aggregate function or select. Other drivers need field in group or aggregate function - not in select.  Would it be possible to merge if I can't write generic test for all drivers?